src/Utils.vala

```diff
@@ -61,4 +61,22 @@ public class Mail.Utils {
     public static string escape_html_tags (string input) {
         return input.replace ("<", "&lt;").replace (">", "&gt;");
     }
+
+    public static string strip_html_tags (string input) {
```

Comment:
If this is only used in one place, it probably doesn't need to be its own public util in another class

Reply:
Removed completely, since its no longer used.